engine/flink/tests/src/test/scala/pl/touk/nussknacker/extensions/SpelExtension.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.extensions
+
+import pl.touk.nussknacker.engine.graph.expression.Expression
+
+trait SpelExtension {
```

Comment:
IMO it is better to use converters via object imports (like scala do with `import scala.jdk.CollectionConverters._` )